src/Hipace.cpp

```diff
@@ -613,7 +619,7 @@ Hipace::ExplicitSolveOneSubSlice (const int lev, const int step,
 
     m_fields.AddRhoIons(lev);
 
-    // deposit jz_beam and maybe rho_beam on This slice
+    // deposit jz_beam and maybe rhomjz on This slice
```

Comment:
```suggestion
    // deposit jz_beam and maybe rhomjz of the beam on This slice
```
I prefer this to be clarified that this is the beam contribution to `rhomjz`